client/postit/src/components/SignIn.js

```diff
@@ -0,0 +1,53 @@
+import 'bootstrap/dist/css/bootstrap.css';
+
+function SignIn(){
+    return <div>
+         <nav class="navbar navbar-dark bg-dark">
+        <h2 class="navbar-text">PostIt</h2>
+       </nav>
+
+       <section class="vh-100" style={{backgroundColor: "#508bfc"}}>
```

Comment:
Could you instead of using this inline style use classes, it looks abit neater that way

Reply:
thanks